EMS/appsettings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "AppSettings":{
```

Comment:
Don't need AppSettings object. You can directly keep variable in root object. Also name it as EmployeesJsonPath